.mergify.yml

```diff
@@ -1,18 +1,62 @@
 # See https://doc.mergify.io
 
 pull_request_rules:
-- name: auto_merge
-  conditions: 
-    - -title~=(WIP|wip)
-    - -merged
-    - -closed
-    - -locked
-    - -label=blocked
-    - -label=do-not-merge
-    - "#approved-reviews-by>=1"
-    - "#changes-requested-reviews-by=0"
-    - status-success=AWS CodeBuild us-east-1 (aws-cdk-build)
-  actions:
-    merge:
-      method: squash
-      strict: smart
+  - name: automatic merge non-sensitive
```

Comment:
remove "non-sensitive". this is currently true for all modules, right?

Reply:
[Fixed!](https://github.com/aws/aws-cdk/commit/8bcab058aae95f86e4689acc51b498799bf06e90)